components/CaseStatus/CaseStatusDetailsTable.tsx

```diff
@@ -9,17 +9,141 @@ interface Props {
   styleType?: string;
   status: CaseStatus;
   answers?: CaseStatusAnswerDisplay;
+  sheduledStatus?: CaseStatusAnswerDisplay[];
```

Comment:
This seems to be misspelt, might be worth fixing but not a blocker

Reply:
Fixed :)